migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,5 @@
+var Storage = artifacts.require('./Storage.sol') // eslint-disable-line no-undef
```

Comment:
undo lint disable here?

Reply:
It won't work then. If you want to fix this before it goes in, ping Jon to do it since it's his code.